Documentation/botr/profiling.md

```diff
@@ -146,9 +146,9 @@ Assembly, Module, Class, Function, and GCHandleIDs have app-domain affinity, mea
 
 All IDs except ObjectID should be treated as opaque values. Most IDs are fairly self-explanatory. A few are worth explaining in more detail:
 
-**ClassIDs** represent classes. In the case of generic classes, they represent fully-instantiated types. List<int>, List<char>, List<object>, and List<string> each have their own ClassID. List<T> is an uninstantiated type, and has no ClassID. Dictionary<string,V> is a partially-instantiated type, and has no ClassID.
+**ClassIDs** represent classes. In the case of generic classes, they represent fully-instantiated types. List\<int>, List\<char>, List\<object>, and List\<string> each have their own ClassID. List\<T> is an uninstantiated type, and has no ClassID. Dictionary\<string,V> is a partially-instantiated type, and has no ClassID.
```

Comment:
Would it be better to use backticks to for these instead?

Reply:
Not sure, I found that GitHub Markdown supports both ways of escaping those.

Reply:
They will render differently. We tend to use the back ticks to escape code fragments, even if it is just complex type name.

One example from many - look for `List<T>` in https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/framework-design-guidelines-digest.md